### DISCUSSION TITLE
Ödev teslimi

### DISCUSSION THREAD

**Discussion by tunahanbucak** [gusanmaz/bmb212-algorithms-2024#2] [Q&A] [answered]
Herkese merhaba,
Leetcode IDE içinde sadece çözüm fonksiyonunun kodu var. Main olmadığı için leetcode harici compile yapılamıyor. Ödevi gönderirken Main metodu dahil edilecek mi? Eğer Main dahil edeceksek Main içine input output kodlarını da yazmalı mıyız?

**Comment by gusanmaz** [comment, marked as answer]
Merhaba Tunahan,

Main metodunu dahil etmeniz gerekmiyor.

**Comment by Amirco-h** [comment]
merhaba, hocam 
bazen konular tekrarlanmış böyle durumlarda  ne yapmalıyız , sade tanesi yaparız yoksada iki tekrar yapılacak , işaret yaptım :
![leetcode-arrays_edit_267434965244086](https://github.com/gusanmaz/bmb212-algorithms-2024/assets/169270369/f9a863dd-ee8e-47a6-b26d-eb60c5494208)

**Comment by gusanmaz** [comment]
Merhaba,

Sorularin icerigi ayni ise, birini cevaplamaniz yeterli. Sorularin icerikleri farkli ise sorulari `Remove-Element-1`, `Remove-Element-2` seklinde sorularin isimlerinin sonuna sira numarasi koyarak isimlendirerek sorularin kodlarini reponuza yukleyebilirsiniz. Bu arada her farkli soru farkli bir baslik olusturmaniz (baskasinin baska bir konuda actigi basliga yorum olarak soruyu yazmaktansa) daha iyi olacaktir.
